src/vs/workbench/browser/parts/editor/editorGroupView.ts

```diff
@@ -551,7 +552,11 @@ export class EditorGroupView extends Themable implements IEditorGroupView {
 		this._onDidGroupChange.fire({ kind: GroupChangeKind.EDITOR_STICKY, editor });
 	}
 
-	private onDidOpenEditor(editor: EditorInput): void {
+	private onDidMoveEditor(event: EditorMoveEvent): void {
+		this._onDidGroupChange.fire({ kind: GroupChangeKind.EDITOR_MOVE, editor: event.editor, previousEditorIndex: event.index, editorIndex: event.target });
+	}
+
+	private onDidOpenEditor(event: EditorOpenEvent): void {
```

Comment:
How about something cool:

```ts
private onDidOpenEditor({ editor, index }: EditorOpenEvent): void {
```

Reply:
Added. I'm not sure why but I've always been a fan of just referencing properties.